pkg/runner/cluster_k8s.go

```diff
@@ -204,6 +204,33 @@ func (*ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow io.Wri
 
 	var gg errgroup.Group
 
+	// TODO: hack - this is the daemon, and not the user's local machine
+	workDir := filepath.Join(input.EnvConfig.WorkDir(), "results")
+	if err := os.MkdirAll(workDir, 0777); err != nil {
+		return nil, err
+	}
+
+	// TODO: hack - create the run output directory and write the runenv.
+	runDir := filepath.Join(workDir, input.RunID)
+	if err := os.MkdirAll(runDir, 0777); err != nil {
+		return nil, err
+	}
+	if f, err := os.Create(filepath.Join(runDir, "env.json")); err == nil {
```

Comment:
Let's make a helper function for this.

Reply:
Sure, this PR is just trying to flesh out the `flow` of `logs`, and when we agree on that, we can refactor this.